osu.Game/Graphics/DrawableDate.cs

```diff
@@ -13,13 +13,59 @@ namespace osu.Game.Graphics
     public class DrawableDate : OsuSpriteText, IHasTooltip
     {
         private readonly DateTimeOffset date;
+        private readonly String dateFormat;
+        private readonly String tooltipFormat;
 
         public DrawableDate(DateTimeOffset date)
         {
             AutoSizeAxes = Axes.Both;
             Font = "Exo2.0-RegularItalic";
 
             this.date = date.ToLocalTime();
+            // if date's format is not specified, set it to an empty string,
+            // so that later we will know to humanize it
+            dateFormat = "";
+            // if tooltip's format is not specified, set it to an empty string
+            // so later we will know to default to a default time format
+            tooltipFormat = "";
+        }
+
+        public DrawableDate(string dateFormat, DateTimeOffset date)
```

Comment:
Instead of 3 new constructors, how about change the one that existed previously to `public DrawableDate(DateTimeOffset date, string dateFormat = null, string tooltipFormat = null)`

With comment on dateformat: "The string to format the date text with. May be null if the humanized format should be used."
And on tooltipFormat: "The string to format the tooltip text with. May be null if the default format should be used."

The comments in the ctor would be unnecessary then.